src/cache/mod.rs

```diff
@@ -576,6 +576,44 @@ impl Cache {
         })
     }
 
+    /// Retrieves a [`Channel`]'s message from the cache based on the channel's and
+    /// message's given Ids.
+    ///
+    /// **Note**: This will clone the entire message.
+    ///
+    /// # Examples
+    ///
+    /// Retrieving the message object from a channel, in a
+    /// [`EventHandler::message`] context:
+    ///
+    /// ```rust,ignore
```

Comment:
We don't need to make this ignored, we can instead do:

```rust
use serenity::CACHE;

# use serenity::model::id::MessageId;
# let message = MessageId(0).get().unwrap();

let cache = CACHE.read();
let fetched_message = ...
```

This way we can at least make it `no_run` and hide the code getting the "message" being used here.

Reply:
Changed with latest commit.